packages/wallet/src/persistence/pouchdbStores/pouchdbWalletStores.ts

```diff
@@ -57,6 +59,7 @@ export const createPouchdbWalletStores = (
           destroyDocumentsDb,
           this.transactions.destroy(),
           this.inFlightTransactions.destroy(),
+          this.volatileTransactions.destroy(),
```

Comment:
This line is not needed: all document stores are destroyed via `destroyDocumentsDb`

Reply:
👍 
Fixed and squashed in one go: https://github.com/input-output-hk/cardano-js-sdk/commit/2a4ccb0abead34481e817f807850d29e77d7340a#diff-47afda8b8869c2531c861dbf59d047a481b4765d18b8e5f8ba0a7b19ada7c02dL59